src/figuresClasses.ts

```diff
@@ -1,11 +1,98 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: string;
+
+  color: string;
+
+  private a: number;
+
+  private b: number;
+
+  private c: number;
+
+  constructor(color: string, a: number, b: number, c: number) {
+    this.shape = 'triangle';
+    this.color = color;
+
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Invalid measurements');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('Invalid measurements');
```

Comment:
[CODE STYLE] - If you throw an error try to provide its detailed description in error message. For example, instead of 'Invalid measurements', specify what kind of invalid measurements, like 'Side lengths must be greater than zero' and 'The sum of the lengths of any two sides must be greater than the length of the third side'.